src/middlewared/middlewared/client/client.py

```diff
@@ -159,7 +165,8 @@ def get_reserved_portfd(self, blacklist=None):
             else:
                 break
         if fd < 0:
-            raise ValueError('Failed to reserv a privileged port')
+            log.debug('Failed to reserve a privileged port')
```

Comment:
We don't need logging here anymore

Reply:
Does the alert system log this somewhere? I think we would still like to know when this is failing somehow, right?

Reply:
We raise exception, it's enough. Caller will log if necessary..